test_load.py

```diff
@@ -0,0 +1,35 @@
+"""Unit tests for load.py"""
```

Comment:
All looks good to me - assuming they pass on your end?

Super minor but maybe we should add a short sentence like 'all tests passing' to descriptions when we do pull requests for tests.